docs/build-ansible.rst

```diff
@@ -196,6 +196,8 @@ Announcing Ansible
 * Change the version numbers.
 * Change the sha256sum
 * Add any info specific to this release.
+  * Send any important information (like one-off changes to the release schedule) from here to
+    `The Bullhorn <https://meet.google.com/eja-xdrg-pqi>`_
```

Comment:
Why not to https://github.com/ansible/community/issues/546 ?

Reply:
Copy paste error, fixed. Also added reddit